EXAMPLES.md

```diff
@@ -349,13 +358,12 @@ To use Apple Pay, you must also enable **Background fetch** in the Player Settin
 3. Under **Other Settings**, set **Behavior in Background** to **Custom**.
 4. Enable **Background fetch**.
 
-### Enabling Apple Pay on your Store
-
-To enable Apple Pay for your store through an app:
+### Enabling native pay on your Store
 
-1. Add the [Mobile App sales channel](https://help.shopify.com/api/sdks/custom-storefront/mobile-buy-sdk/add-mobile-app-sales-channel) in your Shopify admin.
-2. Enable Apple Pay in the Mobile App settings page.
+To enable native pay for your store through an app:
 
+1. Follow [these instructions](https://help.shopify.com/manual/apps/private-apps#generate-credentials-from-the-shopify-admin) to create a private app in your Shopify admin page.
+1. Enable _Allow this app to access your storefront data using the Storefront API_ flag in your private app configuration.
```

Comment:
Should be 2.

Reply:
Actually, this was intentional because 1. is a valid markup for _whatever number this position must be in this enumeration_. I personally like to keep all of them 1s because if we delete, add or reorder items, we don't need to rewrite the numbers to keep them in order.